Source/inv.cpp

```diff
@@ -1601,6 +1602,10 @@ void AutoGetItem(int pnum, Item *item, int ii)
 		}
 	} else {
 		done = AutoEquipEnabled(player, player.HoldItem) && AutoEquip(pnum, player.HoldItem);
+		if (done) {
+			pickupSound = ItemInvSnds[ItemCAnimTbl[item->_iCurs]];

```

Comment:
I think you misunderstood my comment earlier?

Autoequip sounds _already_ play in a separate place. When _picking_ items up, we should play only `IS_IGRAB`

Reply:
> I think you misunderstood my comment earlier?

@julealgon yes indeed, I misunderstood that one, sorry.

I now see that at [inv.cpp#331](https://github.com/diasurgical/devilutionX/blob/a17b0bf947a1a60649de3111868837db2b140f18/Source/inv.cpp#L331) the auto equip sounds are played, so playing those again doesn't make sense of course :see_no_evil: 

In d78e7b4bcbe251c34f9103bd055d38d2d88fc96a I adjusted the code to just play `IS_IGRAB`. I noticed that when picking up an item which gets auto-equipped, both sounds are playing, the item sound and `IS_IGRAB`. Therefore I also adjusted the code to only play the sound when the item wasn't auto-equipped.